pypopquiz/io.py

```diff
@@ -24,8 +24,10 @@ def verify_input(input_data: Dict) -> None:
         "properties": {
             "round": {"type": "number"},
             "theme": {"type": "string"},
-            "spacers": {"type": "boolean"},
+            "spacers": {"type": "string"},
```

Comment:
I had to edit one of my test json files as a result of this edit (changed true to "true"). I assume this is in preparation for a future change where the spacer text can be selected?

Reply:
Oh, I thought that was already the case but you forgot to change this. I'll revert it!